docs/apify_academy/getting_started.md

```diff
@@ -0,0 +1,18 @@
+Internal draft of the structure. Inspired by https://gitlab.com/apify-public/wiki/-/wikis/home
+
+Different categories might be split into more articles. Since we have already pretty detailed documentation about most services, this academy will focus on listing other resources to read and glueing them together with the web scraping academy code tutorials (upgrading the code)
+
+I'm not happy with the top level category names
+
+- Setting up Apify project
+    - Apify CLI (create base project structure)
+    - Apify SDK (upgrading scraping academy code to use the SDK)
```

Comment:
I already have basic SDK code in the web scraping academy. In the advanced and further sections, I expect to use only the SDK.